mods/farming/api.lua

```diff
@@ -343,9 +343,7 @@ function farming.register_plant(name, def)
 	minetest.register_lbm({
 		name = ":" .. mname .. ":start_nodetimer_" .. pname,
 		nodenames = lbm_nodes,
-		action = function(pos, node)
-			tick_again(pos)
-		end,
+		action = tick_again,
```

Comment:
Please don't. See #1114

Reply:
and how is this related?

Reply:
Sorry, what I had in my mind was 626cd8e9c7c2
`tick_again` is not exposed anyway, so this here is fine.